charts/helm-project-operator/templates/rbac.yaml

```diff
@@ -31,93 +28,5 @@ imagePullSecrets: {{ toYaml .Values.global.imagePullSecrets | nindent 2 }}
 # As of now, due to the fact that the k3s-io/helm-controller can only deploy jobs that are cluster-bound to the cluster-admin
 # ClusterRole, the only way for this operator to be able to perform that binding is if it is also bound to the cluster-admin ClusterRole.
 #
-# As a result, this ClusterRole will be left as a work-in-progress until changes are made in k3s-io/helm-controller to allow us to grant
+# As a result, this ClusterRoleBinding will be left as a work-in-progress until changes are made in k3s-io/helm-controller to allow us to grant
 # only scoped down permissions to the Job that is deployed.
-#
-# apiVersion: rbac.authorization.k8s.io/v1
-# kind: ClusterRole
-# metadata:
-#   name: {{ template "helm-project-operator.name" . }}
-#   labels: {{ include "helm-project-operator.labels" . | nindent 4 }}
```

Comment:
This `include` call was causing the same label to appear twice on the above ServiceAccount since each new label was not prepended with a `#`. As a result, you would get the following error:

```bash
$ helm template helm-project-operator charts/helm-project-operator | kubeconform -strict
stdin - ServiceAccount helm-project-operator failed validation: error unmarshalling resource: error converting YAML to JSON: yaml: unmarshal errors:
  line 32: key "app.kubernetes.io/managed-by" already set in map
  line 33: key "app.kubernetes.io/instance" already set in map
  line 34: key "app.kubernetes.io/version" already set in map
  line 35: key "app.kubernetes.io/part-of" already set in map
  line 36: key "chart" already set in map
  line 37: key "release" already set in map
  line 38: key "heritage" already set in map
```

This also meant that supplying `.Values.global.imagePullSecrets` was broken till this was removed, since it would result in arbitrary keys being added after the `imagePullSecrets` (associated with the labels):

```bash
arvindiyengar: ~/Rancher/helm-project-operator/src/github.com/aiyengar2/helm-project-operator 
$ helm template --set 'global.imagePullSecrets[0].name=mySecret' helm-project-operator charts/helm-project-operator | kubeconform --strict
stdin - ServiceAccount helm-project-operator is invalid: For field imagePullSecrets.0: Additional property chart is not allowed - For field imagePullSecrets.0: Additional property heritage is not allowed - For field imagePullSecrets.0: Additional property release is not allowed - For field imagePullSecrets.0: Additional property app.kubernetes.io/instance is not allowed - For field imagePullSecrets.0: Additional property app.kubernetes.io/managed-by is not allowed - For field imagePullSecrets.0: Additional property app.kubernetes.io/part-of is not allowed - For field imagePullSecrets.0: Additional property app.kubernetes.io/version is not allowed
```

This is no longer the case as of this PR.